### ISSUE TITLE
Use existing MYSQL database server

### ISSUE THREAD

**Issue by londonpete1958** [leokhoa/laragon#373]
I have MYSQL with 30+ databases already defined. How can I install and get Laragon to use the existing install, and not another install in the Laragon directory?

This includes using an existing root user and it's password, not a blank password that Laragon seems to favour.

Many thanks

**Comment by Pen-y-Fan** [comment]
Yes, when Laragon has been installed start Apache and your MySQL server. 

Should you accidentally **Start all**, which will start Apache and Laragon's MySQL server, you will likely see a conflict on port 3306. Which is the default port used by MySQL servers.

You can [change the port](https://laragon.org/docs/screenshots.html#Services-amp-Ports) for Laragon's MySQL server, and try 33060 or 33061.

**Comment by londonpete1958** [comment]
Thanks for this, it does seem strange that you cannot pick and choose if you want to use an existing component. I have PHP  already installed, but that's not such a big deal, but still would be cleaner if you could choose. Many thanks PeteSent from my Galaxy
-------- Original message --------From: Pen y Fan ***@***.***> Date: 23/01/2023  18:32  (GMT+00:00) To: leokhoa/laragon ***@***.***> Cc: Pete Harrison ***@***.***>, Author ***@***.***> Subject: Re: [leokhoa/laragon] Use existing MYSQL database server (Issue #373) 
Yes, when Laragon has been installed start Apache and your MySQL server.
Should you accidentally Start all, which will start Apache and Laragon's MySQL server, you will likely see a conflict on port 3306. Which is the default port used by MySQL servers.
You can change the port for Laragon's MySQL server, and try 33060 or 33061.

**Comment by Pen-y-Fan** [comment]
Actually, you can deselect MySQL in Laragon's preferences.

![Deselect MySQL](https://user-images.githubusercontent.com/40126936/214128497-7dde44b9-d53e-4f7b-9eab-18209d093181.png "Deselect MySQL")

**Comment by londonpete1958** [comment]
Using my own install of MYSQL works when accessing the databases, but the one click WordPress fils because it trys to use the root user with a blank password. This is rediculous and a security weakness. Again, why can't I specify the passwordRegards Pete Sent from my Galaxy
-------- Original message --------From: Pen y Fan ***@***.***> Date: 23/01/2023  19:12  (GMT+00:00) To: leokhoa/laragon ***@***.***> Cc: Pete Harrison ***@***.***>, Author ***@***.***> Subject: Re: [leokhoa/laragon] Use existing MYSQL database server (Issue #373) 
Actually, you can deselect MySQL in Laragon's preferences.

**Comment by Pen-y-Fan** [comment]
That is normal, you can't have the penny and the bun 🤣

You want to use Laragon magic with half the magic switched off 🤷

A blank password is typical for local development unless you expose your MySQL server to the internet.

**Comment by eagleuk** [comment]
> Using my own install of MYSQL works when accessing the databases, but the one click WordPress fils because it trys to use the root user with a blank password. This is rediculous and a security weakness. Again, why can't I specify the passwordRegards Pete Sent from my Galaxy -------- Original message --------From: Pen y Fan ***@***.***> Date: 23/01/2023 19:12 (GMT+00:00) To: leokhoa/laragon ***@***.***> Cc: Pete Harrison ***@***.***>, Author ***@***.***> Subject: Re: [leokhoa/laragon] Use existing MYSQL database server (Issue #373) Actually, you can deselect MySQL in Laragon's preferences. —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

Generally speaking, I don't think anyone uses those one click install things anymore.  It's much easier especially for Drupal to kick everything off with composer on the command line.  I believe at some point they whould probably just be removed. 

IMO most people just use quick app > blank.

**Comment by josealgardataborges** [comment]
> IMO most people just use quick app > blank.

Or just plain composer on the command line

**Comment by alxndr-w** [comment]
I will close this issue as the original question was solved: disabling MySQL that comes with Laragon from the settings.

Btw: Another option would have been to export and import a dump from MySQL outside laragon to the DBMS that comes with Laragon.
